tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -1,99 +1,99 @@
+                OrganisationOdsCode,
+                OrganisationAddressId,
+                OrganisationBillingAddressId,
+                OrganisationContactId,
+                OrderStatusId,
+                Created,
+                LastUpdated,
+                LastUpdatedBy,
+                LastUpdatedByName
+            )
+            VALUES
+            (
+                @OrderId,
+                @Description,
+                @OrganisationId,
+                @OrganisationName,
+                @OrganisationOdsCode,
+                @OrganisationAddressId,
+                @OrganisationBillingAddressId,
+                @OrganisationContactId,
+                @OrderStatusId,
+                @Created,
+                @LastUpdated,
+                @LastUpdatedBy,
+                @LastUpdatedByName
+            );";
+
         public static async Task<string> FetchOrderDescriptionFromOrderId(string connectionString, string orderId)
-        {
-            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
-                         [Description]
-                         FROM [Order]
+        {
+            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
+                         [Description]
+                         FROM [Order]
+                         WHERE [OrderId] = @orderId", new { orderId }));
+        }
+
+        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
+        {
+            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
+                         [LastUpdatedByName]
+                         FROM [Order]
                          WHERE [OrderId] = @orderId", new { orderId }));
-        }
-
-        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
-        {
-            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
-                         [LastUpdatedByName]
-                         FROM [Order]
-                         WHERE [OrderId] = @orderId", new { orderId }));
-        }
-
+        }
+
         public static async Task<OrderEntity> FetchOrderByOrderId(string connectionString, string orderId)
         {
-            return (await SqlRunner.QueryFirstAsync<OrderEntity>(connectionString, $@"SELECT
-                          [OrderId],
-                          [Description],
-                          [OrganisationId],
-                          [OrderStatusId],
-                          [Created],
-                          [LastUpdated],
-                          [LastUpdatedBy],
-                          [LastUpdatedByName]
-                         FROM [Order]
-                         WHERE [OrderId] = @orderId", new { orderId }));
-        }
-    }
-}
+            return (await SqlRunner.QueryFirstAsync<OrderEntity>(connectionString, $@"SELECT

```

Comment:
As we're not actually doing any string interpolation here we can omit the `$`.

Reply:
👍